pages/docs/form/form-control.mdx

```diff
@@ -58,6 +69,38 @@ import {
 </FormControl>
 ```
 
+### Error message
+
+`FormErrorMessage` will only show up when the property `isInvalid` in
+`FormControl` is true.
+
+```jsx
+function errorMessageExample() {
+  const [input, setInput] = useState('')
+
+  const handleInputChange = (e) => setInput(e.target.value)
+
+  const isError = input === ''
+
+  return (
+    <FormControl isInvalid={isError}>
+      <FormLabel htmlFor='email'>Tell me how you love chakra-ui 💛</FormLabel>
```

Comment:
Let's turn this into the old "Email" input.

Label: "Email"
Detail: "Enter the email you'd like to receive the newsletter on."
Error: "Email is required."

Also, we should add a ternary conditional rendering for the error and detail:

```typescript
{isError ? (
   <FormErrorMessage>Email is required.</FormErrorMessage>
) : (
   <FormHelperText>Enter the email you'd like to receive the newsletter on.</FormHelperText>
)}
```

Reply:
I think the `FormErrorMessage` is triggerred by `isInvalid` prop and this is what I'm trying to tell users here.
Or maybe I should remove `FormHelperText`.

Reply:
No, keep it! You're demonstrating a very common use case with that.